libs/core/tests/unit_tests/prompts/test_chat.py

```diff
@@ -641,6 +644,105 @@ async def test_chat_tmpl_from_messages_multipart_image() -> None:
     assert messages == expected
 
 
+async def test_chat_tmpl_from_messages_multipart_formatting_with_path() -> None:
+    """Verify that we can pass `path` for an image as a variable."""
+    base64_image = "/9j/4AAQSkZJRgABAQEASABIAAD/2wBDAAEBAQEBAQEBAQEB"
+    other_base64_image = "/9j/4AAQSkZJRgABAQEASABIAAD/2wBDAAEBAQEBAQEB"
+
+    image_data = base64.b64decode(base64_image)
+
+    with tempfile.NamedTemporaryFile(delete=True, suffix=".jpg") as temp_file:
```

Comment:
@thdesc I changed the delete=True so that clean up is immediate and created a separate unit test

Reply:
@eyurtsev Better this way you are right ! Thanks for the review